languages/javascript/concepts/errors/about.md

```diff
@@ -1 +1,67 @@
-TODO: add information on errors concept
+Errors are useful to report when something is wrong or unexpected in a program or a piece of code.
+
+They are javascript objects.
+
+The main property of this object is `message`:
+
+```javascript
+const error = new Error('Oops, something went wrong')
+
+console.log(error.message)
+// => "Oops, something went wrong"
+```
+
+Using the `throw` syntax, you can throw an Error.
+
+```javascript
+throw new Error('Oops')
+```
+
+When an Error is thrown, the current execution is stopped and resume in the first catch block of the call stack.
+
+```javascript
+try {
+  throw new Error('Oops')
+} catch (error) {
+  console.log(error.message)
+  // => "Oops"
+}
+```
+
+As any object in Javascript the Error can be "extended" to create Custom errors. You can use the `instanceof` syntax to check if the error catched is an instance of a particular object.
+
+```javascript
+class CustomError extends Error {}
+
+try {
+  // ... Code that may throw an error
+} catch (error) {
+  if (error instanceof CustomError) {
+    console.log('The error thrown is an instance of the CustomError')
+  }
+}
+```
+
+## Error Types
+
+In addition to the `Error` object, other builtin error objects exist. You can learn more about it [here][error-types]
+
+## Custom Errors
+
+You can also define your own [Custom error Type][custom-error-type] by creating a class that extends one of the builtins Error Types
```

Comment:
`builtins` -> `built-in`

Reply:
aaa7fe2a557b19ad60f0a2626d6e2f7b734d1f0c